.github/workflows/windows-bundler.yml

```diff
@@ -23,10 +23,10 @@ jobs:
         ruby:
           - { name: jruby-9.2, value: jruby-9.2.16.0 }
           - { name: ruby-2.4, value: 2.4.10 }
-          - { name: ruby-2.5, value: 2.5.8 }
-          - { name: ruby-2.6, value: 2.6.6 }
-          - { name: ruby-2.7, value: 2.7.2 }
-          - { name: ruby-3.0, value: 3.0.0 }
+          - { name: ruby-2.5, value: 2.5.9 }
+          - { name: ruby-2.6, value: 2.6.7 }
+          - { name: ruby-2.7, value: 2.7.3 }
+          - { name: ruby-3.0, value: 3.0.1 }
```

Comment:
You could potentially use `'2.5'`/`'2.6'`/etc and then it would automatically update when RubyInstaller has a build for it (and I run a script in setup-ruby). Depends if you want Ruby patch updates to be explicit or not.

Reply:
Yeah, we're aware, thanks! I do prefer explicit patch levels. I understand auto-selection can be handy for many users, but for us I prefer explicit, given how hard our CI is to keep green, and how deeply connected with core ruby rubygems & bundler are.